modules/ROOT/pages/govern-api-instances.adoc

```diff
@@ -0,0 +1,45 @@
+= Governing API Instances
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+API Manager is closely integrated with xref:api-governance::index.adoc[API Governance] to govern aspects of API instances with governance rulesets. Applying a governance ruleset to your API ensures that every instance of that API conforms to standards such as what policies are applied and if a TLS context is required.  
+
+To govern an API instances, see <<apply>>.
+
+After applying a Governance ruleset to a specific API, view and fix the conformance issues for its API instances:
+
+* <<view-api-manager>>
+* <<view-api-governance>>
+
+[[apply]]
+== Apply a Governance Rulesets
+
+To govern API instances, you must apply a governance ruleset to your desired API. To apply a governance ruleset, see xref:api-governance::create-profiles.adoc[].
+
+Additionally, you can create custom rulesets to satisfy your specific needs. To create custom governance rulesets, see xref:api-governance::create-custom-rulesets.adoc[].
```

Comment:
Since they will definitely need to create their own custom rulesets for governing instances if they want to really get value out of Anypoint API Governance, I suggest something a little more prescriptive:
Create custom rulesets to add rules specific to your own API management standards. See xref:api-governance::create-custom-rulesets.adoc[].